242 disable autoconf

maxjmohr opened the pull request:
### Description

Update documentation for new start and connect scripts

## Testing Instructions

Check if documentation is inline with scripts

### Type of Change

Please delete options that are not relevant.

- [ ] 🐛 Bug fix
- [ ] ✨ New feature
- [x] 🧹 Code cleanup or refactor

maxjmohr commented on doc/Admin Guide/Setup TruSpace/Connecting to other nodes.md:
```diff
@@ -1,43 +1,114 @@
+<br>
+<details>
+<summary>To get the peer ip and peer ids, you can use this script</summary>
+On the target node, use the following script to obtain relevant identifier values that you need to connect your source node to the target node:
```
I couldnt find the script, did you push it?

jzakotnik replied:
I can't figure out why it disappeared, it's still here: https://github.com/openkfw/TruSpace/compare/152de27a1195501b695e8c0a8e275cca22013842...33fdcb4c02c0ab7cb1527f1a66b61ad9f73b4529

maxjmohr commented on README.md:
````diff
@@ -74,107 +59,65 @@ If something doesn't work, check that all containers are running with `docker ps
 
 ```bash
-./connectPeer.sh 213.154.217.25 <IPFS PeerID> <Cluster PeerID>
+./connectPeer.sh <peer_ip> <ipfs_peer_id> <cluster_peer_id> <ipfs_container_id> <cluster_container_id> [swarm_key_path] [cluster_secret_path]
````
Ah now I understand. Yes I think it makes sense because 1) as you said, it might be the case that you have stored the key in e.g. your Downloads folder as have received it from some one. Same goes for the cluster secret, e.g. as a .txt file.
The 2) reason I think it's nice is because the admin/dev doesn't have to think where exactly to insert the keys but the script does it automatically (target location might not be clear for everyone).
So for cluster secret, I read it from a file (e.g. .txt) and then insert it into the service.json as the secret. But I don't copy the entire service.json, only the secret. Hope this is understandable? Will update the guide to clarify

maxjmohr replied:
Tried to clarify in [5812a6d](https://github.com/openkfw/TruSpace/pull/247/commits/5812a6dd2bc97a429910ca24c0c3a5992a9c61c2)

maxjmohr replied:
I like that idea a lot! Will try to write such a script that outputs all the relevant connection settings in a directory or something similar

maxjmohr commented:
@jzakotnik some minor and major updates
- Relocated some files into the script folder and updated their paths if they were mentioned/used somewhere
- Renamed `connectPeer.sh` into `connectPeer-manually.sh` so one can still manually input the connection variables and run the script
- Added `fetch_connection.sh` so a user can fetch the relevant connection variables on their local device. These values are stored in a `.connection` file, without encryption so the user can have a look. If the user wants to send this file to the other person (who wants to connect to their nodes), they can add an email as an input. Then, the `.connection` becomes encrypted with a password, which itself is stored in `.connection.password`. 2 emails are automatically generated with some instructions; the user adds both files to their respective email and sends it to the other IPFS colleague.
- Added `connectPeer-automatic.sh` which only takes `.connection` and `.connection.password` as inputs. It automatically encrypts the files, reads the connection variables and ingests them into the right places`. It calls `connectPeer-manually.sh` so if we want/have to change something in the ingestion, it shouldnt be too much duplicated code.

This should really streamline the connection process into only a few steps:
A has the connection, B wants to connect to A:
1. A runs `bash fetch_connection.sh B@Borganisation.com` and sends 2 emails with the connection files to B
2. B downloads files from emails into root project directory
3. B runs `bash connectPeer-automatic.sh .connection .connection.password`
4. Nodes should now be connected!

I have also screen recorded the workflow, have a look:
https://github.com/user-attachments/assets/4fa39277-b948-4d22-8ef6-bb219f42da9b

Please have a look at the scripts and workflow and let me know your thoughts.
If you like this as is, I will update the doc and README to incorporate the new scripts.
